proposals/xxxx_carvel-package/README.md

```diff
@@ -0,0 +1,231 @@
+via `PackageInstall` custom resources, and `Secret`s containing `Package` configuration.
+
+Conceptually, cluster managers would make the Pinniped software available on the 
+cluster by deploying the PackageRepository:
+
+```bash
+# Deploy the Pinniped PackageRepository to the globally available
+# namespace watched by kapp-controller for new Packages 
+kapp deploy --app pinniped-package-repository --file <pinniped-release-files>/pinniped-package-repository.yaml
+```
+
+Then developers responsible for deploying Supervisor and Concierge would create the 
+appropriate resources to successfully deploy the PackageInstall and Packages for both 
+`Supervisor` and `Concierge`:
+
+```bash 
+# create a Service account and RBAC for the PackageInstall 
+vim supervisor-service-and-rbac.yaml
+kapp deploy --app supervisor-rbac --file supervisor-service-and-rbac.yaml
+vim concierge-service-and-rbac.yaml
+kapp deploy --app concierge-rbac --file concierge-service-and-rbac.yaml
+
+# create a PackageInstall and a Secret for configuring the Concierge
+vim supervisor-package-install-bundle.yaml
+kapp deploy --app supervisor --file supervisor-package-install-bundle.yaml
+vim concierge-package-install-bundle.yaml
+kapp deploy --app supervisor --file concierge-package-install-bundle.yaml
+```
+
+The `PackageRepository` will contain a series of versions of each of the Packages for Supervisor 
+and Concierge.
+
+The `PackageInstall` files will contain `constraints` representing acceptable versions of both the 
+`Supervisor` and `Concierge. For example:
+
+```yaml
+spec:  
+  packageRef:
+    # there will be two separate PackageInstall files, one for each 
+    # Supervisor and Concierge
+    refName: "supervisor.pinniped.dev"
+    versionSelection:
+      # Constraints may be used to specify an exact version of the package    
+      constraints: "0.25.0"  
+      # Alternatively, a constraint can be based on a semver range and can 
+      # specify multiple acceptible versions of the software.  In this case, 
+      # the Package will automatically upgrade to new versions when they become
+      # available, for example, when a new verison of the PackageRepository is 
+      # deployed containing new versions of the Packages.
+      constraints: ">0.25.0"  
+```
+
+### Goals and Non-goals
+
+Goals
+- Provide an additional deployment option to deliver Pinniped software to a Kubernetes cluster
+  in the form of the `Package` apis provided by the Carvel toolchain.
+- Provide a `PackageRepository` and two separate `Package`s for Supervisor and Concierge.
+
+Non-Goals
+- Provide additional deployment alternatives, such as official Helm charts
+- Provide a single package for both Supervisor and Concierge.
+- Provide Packages for testing tools, such as `local-user-authenticator`.
+
+#### API Changes
+
+No changes or additions to Pinniped's own APIs, this proposal represents a second, alternative 
+method for deployment utilising Carvel APIs and tools.  
+
+#### Upgrades
```

Comment:
Could you include a note here about upgrading from a non-Carvel installation to a Carvel installation?

Reply:
I can.  I think it would simply be backup your specific configuration, delete Pinniped & reinstall.